advertools/spider.py

```diff
@@ -949,9 +951,14 @@ def crawl(url_list, output_file, follow_links=False,
             else:
                 setting = '='.join([key, str(val)])
             settings_list.extend(['-s', setting])
+    if meta:
```

Comment:
I think we also need to check `if "proxy" in meta` in addition to `if meta`, otherwise the following line will raise a `KeyError` if the key doesn't exist.